### PR TITLE
Fix messages and comments in telco5g job

### PR DESCRIPTION
**Pull request by sshnaidm**

### CODE REVIEW

File: ci-operator/step-registry/telco5g/cluster-setup/telco5g-cluster-setup-commands.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@ set -o errexit
 set -o pipefail
 
 echo "************ telco cluster setup command ************"
-# TODO: Remove once OpenShift CI will be upgraded to 4.2 (see https://access.redhat.com/articles/4859371)
+# Fix user IDs in a container
```
Comment (yuvalk) [inline ci-operator/step-registry/telco5g/cluster-setup/telco5g-cluster-setup-commands.sh:8]: cant we get rid of it now?

Comment (sshnaidm) [reply ci-operator/step-registry/telco5g/cluster-setup/telco5g-cluster-setup-commands.sh:8]: Nope. Still doesn't work w/o this.

**Comment by yuvalk** [comment]
But you removed the todo ..

**Pull request merged**